youtube_dl/extractor/bigflix.py

```diff
@@ -0,0 +1,54 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from base64 import b64decode
+
+from .common import InfoExtractor
+from ..compat import compat_urllib_parse_unquote
+
+
+class BigflixIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?bigflix\.com/.*/(?P<id>[0-9]+)'
+    _TESTS = [
+        {
+            'url': 'http://www.bigflix.com/Hindi-movies/Action-movies/Singham-Returns/16537',
+            'md5': 'ec76aa9b1129e2e5b301a474e54fab74',
+            'info_dict': {
+                'id': '16537',
+                'ext': 'mp4',
+                'title': 'Singham Returns',
+                'description': 'md5:3d2ba5815f14911d5cc6a501ae0cf65d',
+            }
+        },
+        {
+            'url': 'http://www.bigflix.com/Tamil-movies/Drama-movies/Madarasapatinam/16070',
```

Comment:
If both tests test the same functionality leave only one. Otherwise put short comment for additional test.